packages/dialtone-vue2/components/rich_text_editor/rich_text_editor.vue

```diff
@@ -366,6 +364,7 @@ export default {
                 Enter: () => true,
                 'Shift-Enter': () => this.editor.commands.first(({ commands }) => [
                   () => commands.newlineInCode(),
+                  () => commands.splitListItem('listItem'),
```

Comment:
This line seems to be breaking Shift + Enter when `allowBulletList` is false. This would probably be fixed by adding this action only if `allowBulletList` is true.

Reply:
Thanks for catching that! fixing.